apps/examples/package.json

```diff
@@ -31,18 +31,17 @@
 		"build": "vite build",
 		"lint": "yarn run -T tsx ../../scripts/lint.ts"
 	},
-	"devDependencies": {
-		"@babel/plugin-proposal-decorators": "^7.21.0",
-		"lazyrepo": "0.0.0-alpha.26",
-		"vite": "^4.3.4"
-	},
 	"dependencies": {
+		"@babel/plugin-proposal-decorators": "^7.21.0",
```

Comment:
These seemed correct previously, why are they now `dependencies` rather than `devDependencies`?

Reply:
The dependencies / devDependencies here really don't really matter, as this is not published to npm